README.md

```diff
@@ -223,6 +235,7 @@ Each command can be executed with `invoke <command>`. Each command also has its
   pylint             Run pylint code analysis.
   pydocstyle         Run pydocstyle to validate docstring formatting adheres to NTC defined standards.
   pytest             Run pytest for the specified name and Python version.
+  pytest-without-optionals   This will run pytest only to assert the correct errors are raised when optional dependencies are not installed.
```

Comment:
Can we make it, pytest and pytest-all? Would need to flip the logic though of course.

Reply:
meaning `pytest` is without options, all is with them?

Reply:
yes.